src/components/ui/BaseDialog/BaseDialog.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+	<BaseModal :active.sync="isActive">
+		<div class="modal-header">
+			<h3>{{ title }}</h3>
+		</div>
+
+		<div class="modal-body">
+			<!-- eslint-disable-next-line vue/no-v-html -->
+			<p v-html="message" />
```

Comment:
Why are you using v-html here? You said the use case of this is a simple confirmation message, so why allow html here and may create a xss security vulnerability?
```suggestion
			<p>{{ message }}</p>
```

Reply:
The reason is, that sometimes we want to style the message a bit for example with some <b>bold words</b> in it. I think there is no xss security issue because the message is passed only inside the code and not from an input field etc. Also it's possible to sanitize the html. there is a package for that: sanitize-html
We could use this for the variables whenever we need to use v-html.

Reply:
Most of the time, that's correct. But think about something like `do you wan't to delete task "${task.title}"`. This is a common way to introduce user generated content into the dialog which could be mailcious. Ideally this should be filtered on the server side, but I would prefer the safe way. The package you recommend sound's good.

Reply:
Hmm yeah that could be possible. Let's not use the v-html then for now.

If the content is too complex, we could pass a pseudo-component into the content section, which can have HTML and also logic. I'll make another pull request for that.